remix-solidity/src/analysis/modules/staticAnalysisCommon.js

```diff
@@ -520,6 +520,17 @@ function isPayableFunction (node) {
   )
 }
 
+/**
+ * True if is constructor
+ * @node {ASTNode} some AstNode
+ * @return {bool}
+ */
+function isConstructor (node) {
```

Comment:
I would have expected such a helper to be already present. How are constructors handled in other parts of this code?

Reply:
Handled in what sense? I searched in `remix-solidity` and found only one other reference to `isConstructor` in `src/decoder/internalCallTree.js`.

Reply:
Helper function for the staticAnalysis are in staticAnalysisCommon.js (this file).
Not really sure but it's totally possible that we did not have to handled ctr in static analysis before.
The other `isConstructor` in `internalCallTree`  come from the AST.

Reply:
ok!